src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -35,6 +35,12 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
+// @todo #93:30min NHD needs to be tested against the following after #103 is
+//  fixed: NoMethods, OneVoidMethodWithoutParams, WithoutAttributes,
+//  OneMethodCreatesLambda. NHD score for all these is "NaN".
+// @todo #93:30min NHD calculation needs to take into account the method's
+//  visibility, which should be configurable and implemented after #101 is
+//  fixed.
 /**
  * Tests for all metrics.
```

Comment:
@llorllale The 2 puzzles from above can be in this JavaDoc, Starting at this line.

Reply:
@amihaiemil this is tricky because qulice starts to complain a lot. Starting with `First sentence in a comment should start with a capital letter (RegexpMultilineCheck)`. I'd need to disable some checks on this file; conversely, someone needs to remember to remove the checks when they complete the puzzle.

Do you feel strongly about having them in the javadoc?

Reply:
@llorllale I see, ok